app/assets/stylesheets/finder_frontend.scss

```diff
@@ -290,13 +290,10 @@
         margin-top:-2px;
       }
 
-      h3 {
+      a {
```

Comment:
Could this be a specific class? (also other examples of this below).

Reply:
Thanks - think I've addressed this but might be worth double checking the finders to make sure the styling still looks as it should 🙂